android/CMakeLists.txt

```diff
@@ -1,27 +1,66 @@
 cmake_minimum_required(VERSION 3.9.0)
 
 project(rnscreens)
+set(CMAKE_CXX_STANDARD 20)
 
+if(${IS_NEW_ARCHITECTURE_ENABLED})
+    string(APPEND CMAKE_CXX_FLAGS " -DRCT_NEW_ARCH_ENABLED")
+endif()
```

Comment:
Why are we setting this? Is that because of some unmerged changes earlier for matching RN 0.74?